CHANGELOG.rst

```diff
@@ -20,6 +20,8 @@ This document describes changes between each past release.
 - Server now returns 415 error response if client cannot accept JSON response (#461, mozilla-services/cliquet#667)
 - Server now returns 415 error response if client does not send JSON request (#461, mozilla-services/cliquet#667)
 - Add the ``__lbheartbeat__`` endpoint, for load balancer membership test.
+- Added the ``flush_endpoint``, ``schema`` and ``default_bucket`` capabilities
```

Comment:
> what's inside the resources, but how to access them, and the mean of access is exactly the same as before.

I mean the protocol of the home page changed and how we are getting the information of it ``default_bucket``, ``flush_endpoint`` and ` schema`` are activated